### DISCUSSION TITLE
conditionally-skipping-a-trigger-job-in-gitlab-based-on-a-previous-job/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#422] [Announcements]
# conditionally-skipping-a-trigger-job-in-gitlab-based-on-a-previous-job/

In this post I describe the approaches I explored to conditionally skip a trigger job in GitLab based on a previous job using dynamic pipelines

https://andrewlock.net/conditionally-skipping-a-trigger-job-in-gitlab-based-on-a-previous-job/

**Comment by Arugin** [comment]
Nice
